From: "Mike Rapoport (Microsoft)" <rppt@kernel.org>
Subject: mm/kmemleak: fix typo in object_no_scan() comment
Date: Thu, 10 Oct 2024 18:54:39 +0300

Replace "corresponding to the give pointer" with "corresponding to the
given pointer"

Link: https://lkml.kernel.org/r/20241010155439.554416-1-rppt@kernel.org
Signed-off-by: Mike Rapoport (Microsoft) <rppt@kernel.org>
Cc: Mike Rapoport (Microsoft) <rppt@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/kmemleak.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/kmemleak.c~mm-kmemleak-fix-typo-in-object_no_scan-comment
+++ a/mm/kmemleak.c
@@ -1011,7 +1011,7 @@ static void object_set_excess_ref(unsign
 }
 
 /*
- * Set the OBJECT_NO_SCAN flag for the object corresponding to the give
+ * Set the OBJECT_NO_SCAN flag for the object corresponding to the given
  * pointer. Such object will not be scanned by kmemleak but references to it
  * are searched.
  */
_
